keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/api/ec2/DeserializationInjectionTests.kt

```diff
@@ -16,18 +15,14 @@ import strikt.assertions.isEqualTo
 import strikt.assertions.isNotNull
 import strikt.assertions.isSuccess
 
+@SpringBootTest(
+  classes = [Ec2JsonTestConfiguration::class],
+  webEnvironment = NONE
+)
```

Comment:
Example of replacing custom configuration of the `ObjectMapper` with Spring auto-wiring in tests.